vaccine_feed_ingest/runners/al/arcgis/fetch.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import pathlib
+import sys
+
+# import arcgis ingestor
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
```

Comment:
I'll see if I can so this in a better way in a later PR. I think if I pass `PYTHONPATH=vaccine-feed-ingest` when I call the script then the script authors won't need to setup the import path manually.